src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Assemblies/RoAssembly.GetForwardedTypes.cs

```diff
@@ -49,7 +49,7 @@ public sealed override Type[] GetForwardedTypes()
                         {
                             exceptions = new List<Exception>();
                         }
-                        exceptions.Add(exception);
+                        exceptions.Add(exception!);
```

Comment:
Could have declared `List<Exception?>? exceptions`

Reply:
The `exception` in this scope must be non-null on account of the assertion on line 39.